Travel Scheduler/View Controllers/HomeCollectionViewController.m

```diff
@@ -266,22 +341,24 @@ - (void)makeSchedule
     if(self.arrayOfSelectedPlaces.count > 0) {
         [[[[self.tabBarController tabBar]items]objectAtIndex:1]setEnabled:TRUE];
         ScheduleViewController *destView = (ScheduleViewController *)[[self.tabBarController.viewControllers objectAtIndex:1] topViewController];
-        bool isFirstSchedule = NO;
-        if(destView.selectedPlacesArray == nil) {
-            isFirstSchedule = YES;
-        }
         destView.selectedPlacesArray = self.arrayOfSelectedPlaces;
         destView.regenerateEntireSchedule = true;
         destView.home = self.home ? self.home : self.hub;
         destView.hub = self.hub;
-        if(!isFirstSchedule) {
+        if(destView.selectedPlacesArray != nil) {
             [destView scheduleViewSetup];
         }
+        self.isScheduleUpToDate = YES;
+        self.hasFirstSchedule = YES;
```

Comment:
This is okay for now, but I think later when we merge this with my branch, we should change the hasFirstSchedule property to YES in the schedule view controller because it's possible that the user tries to schedule something that doesn't work even after these initial checks so the first schedule may not necessarily be generated yet.